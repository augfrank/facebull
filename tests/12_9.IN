M0   C0   C1  2000
M1   C0   C2  2064
M2   C0   C3  2462
M3   C0   C4  2560
M4   C0   C5  2492
M5   C0   C6  2476
M6   C0   C7  2312
M7   C0   C8  2582
M8   C0   C9  2656
M9   C0   C10  2399
M10   C0   C11  2589
M11   C1   C0  2927
M12   C1   C2  2431
M13   C1   C3  2577
M14   C1   C4  2831
M15   C1   C5  2014
M16   C1   C6  2966
M17   C1   C7  2300
M18   C1   C8  2192
M19   C1   C9  2917
M20   C1   C10  2547
M21   C1   C11  2764
M22   C2   C0  2871
M23   C2   C1  2372
M24   C2   C3  2521
M25   C2   C4  2087
M26   C2   C5  2909
M27   C2   C6  2677
M28   C2   C7  2584
M29   C2   C8  2440
M30   C2   C9  2733
M31   C2   C10  2909
M32   C2   C11  2712
M33   C3   C0  2600
M34   C3   C1  2619
M35   C3   C2  2744
M36   C3   C4  2763
M37   C3   C5  2959
M38   C3   C6  2862
M39   C3   C7  2069
M40   C3   C8  2069
M41   C3   C9  2514
M42   C3   C10  2974
M43   C3   C11  2044
M44   C4   C0  2673
M45   C4   C1  2824
M46   C4   C2  2007
M47   C4   C3  2067
M48   C4   C5  2710
M49   C4   C6  2827
M50   C4   C7  2357
M51   C4   C8  2299
M52   C4   C9  2038
M53   C4   C10  2660
M54   C4   C11  2930
M55   C5   C0  2982
M56   C5   C1  2514
M57   C5   C2  2323
M58   C5   C3  2123
M59   C5   C4  2029
M60   C5   C6  2109
M61   C5   C7  2906
M62   C5   C8  2038
M63   C5   C9  2065
M64   C5   C10  2350
M65   C5   C11  2149
M66   C6   C0  2675
M67   C6   C1  2647
M68   C6   C2  2760
M69   C6   C3  2018
M70   C6   C4  2042
M71   C6   C5  2028
M72   C6   C7  2574
M73   C6   C8  2217
M74   C6   C9  2768
M75   C6   C10  2014
M76   C6   C11  2759
M77   C7   C0  2350
M78   C7   C1  2453
M79   C7   C2  2809
M80   C7   C3  2167
M81   C7   C4  2844
M82   C7   C5  2993
M83   C7   C6  2805
M84   C7   C8  2846
M85   C7   C9  2755
M86   C7   C10  2324
M87   C7   C11  2626
M88   C8   C0  2801
M89   C8   C1  2606
M90   C8   C2  2739
M91   C8   C3  2969
M92   C8   C4  2034
M93   C8   C5  2991
M94   C8   C6  2322
M95   C8   C7  2859
M96   C8   C9  2789
M97   C8   C10  2797
M98   C8   C11  2961
M99   C9   C0  2711
M100   C9   C1  2469
M101   C9   C2  2305
M102   C9   C3  2935
M103   C9   C4  2212
M104   C9   C5  2782
M105   C9   C6  2087
M106   C9   C7  2339
M107   C9   C8  2646
M108   C9   C10  2805
M109   C9   C11  2212
M110   C10   C0  2155
M111   C10   C1  2118
M112   C10   C2  2635
M113   C10   C3  2769
M114   C10   C4  2282
M115   C10   C5  2817
M116   C10   C6  2004
M117   C10   C7  2183
M118   C10   C8  2379
M119   C10   C9  2952
M120   C10   C11  2271
M121   C11   C0  2212
M122   C11   C1  2855
M123   C11   C2  2484
M124   C11   C3  2547
M125   C11   C4  2458
M126   C11   C5  2966
M127   C11   C6  2151
M128   C11   C7  2770
M129   C11   C8  2631
M130   C11   C9  2045
M131   C11   C10  2571
