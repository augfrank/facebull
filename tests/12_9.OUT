25189
0 15 28 43 46 62 75 80 92 105 110 130
